Products/CMFPlone/browser/configure.zcml

```diff
@@ -32,6 +32,18 @@
       permission="zope.Public"
       />
 
+ <browser:resource
+       file="static/favicon.ico"
+       name="favicon.ico"
+       />
+
+    <browser:page
```

Comment:
Note that favicons can come in a single file multiple sizes.  This is the customary way of doing it.

Additionally, the Plone default favicon at `Plone/favicon.ico` is not themed by Barceloneta, and it is the icon used for many RSS readers (they just default to loading that favicon, because how else would they know what to load, the RSS file does not carry the favicon location metadata).  This favicon can only be themed reliably using a CMFPlone Skin (diff for policy product to do that goes as follows):


```
commit 1f228aacf577ff7974107012e2a44c60f1c70b31
Author: Manuel Amador (Rudd-O) <rudd-o@rudd-o.com>
Date:   Wed Dec 29 02:47:50 2021 +0000

    Add favicon skin.

diff --git a/src/manuelamador.policy/src/manuelamador/policy/configure.zcml b/src/manuelamador.policy/src/manuelamador/policy/configure.zcml
index cb66e86..de012e3 100644
--- a/src/manuelamador.policy/src/manuelamador/policy/configure.zcml
+++ b/src/manuelamador.policy/src/manuelamador/policy/configure.zcml
@@ -3,6 +3,7 @@
     xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
     xmlns:i18n="http://namespaces.zope.org/i18n"
     xmlns:plone="http://namespaces.plone.org/plone"
+    xmlns:cmf="http://namespaces.zope.org/cmf"
     i18n_domain="manuelamador.policy">
 
   <i18n:registerTranslations directory="locales" />
@@ -11,6 +12,13 @@
 
   <include file="permissions.zcml" />
 
+  <!-- We use this to override /favicon.ico -->
+  <cmf:registerDirectory
+      name="skins"
+      directory="skins"
+      recursive="True"
+      />
+
   <genericsetup:registerProfile
       name="default"
       title="manuelamador.name policy"
@@ -38,6 +46,15 @@
       profile="manuelamador.policy:default"
       />
 
+  <genericsetup:upgradeDepends
+      title="Setup favicon skin"
+      description="Skins: add skin for raw favicon"
+      source="*"
+      destination="1016"
+      import_steps="skins"
+      profile="manuelamador.policy:default"
+      />
+
   <utility
       factory=".setuphandlers.HiddenProfiles"
       name="manuelamador.policy-hiddenprofiles"
diff --git a/src/manuelamador.policy/src/manuelamador/policy/profiles/default/metadata.xml b/src/manuelamador.policy/src/manuelamador/policy/profiles/default/metadata.xml
index c49ae57..7e0aa40 100644
--- a/src/manuelamador.policy/src/manuelamador/policy/profiles/default/metadata.xml
+++ b/src/manuelamador.policy/src/manuelamador/policy/profiles/default/metadata.xml
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <metadata>
-  <version>1015</version>
+  <version>1016</version>
   <dependencies>
     <dependency>profile-plone.app.dexterity:default</dependency>
     <dependency>profile-plone.app.relationfield:default</dependency>
diff --git a/src/manuelamador.policy/src/manuelamador/policy/profiles/default/skins.xml b/src/manuelamador.policy/src/manuelamador/policy/profiles/default/skins.xml
new file mode 100644
index 0000000..cb31dfc
--- /dev/null
+++ b/src/manuelamador.policy/src/manuelamador/policy/profiles/default/skins.xml
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<object name="portal_skins" meta_type="Plone Skins Tool">
+ <object name="plone_images_themed" meta_type="Filesystem Directory View"
+         directory="manuelamador.policy:skins/plone_images_themed"/>
+  <skin-path name="*">
+    <layer name="plone_images_themed" insert-after="custom"/>
+  </skin-path>
+</object>
\ No newline at end of file
diff --git a/src/manuelamador.policy/src/manuelamador/policy/skins/plone_images_themed/favicon.ico b/src/manuelamador.policy/src/manuelamador/policy/skins/plone_images_themed/favicon.ico
new file mode 120000
index 0000000..cf60347
--- /dev/null
+++ b/src/manuelamador.policy/src/manuelamador/policy/skins/plone_images_themed/favicon.ico
@@ -0,0 +1 @@
+../../theme/barceloneta-favicon.ico
\ No newline at end of file
```

Reply:
Regarding the other icons, these are more alternatives to the logo i guess.
These are still part of the theme.
I think a better approach would be to allow the configuration of a manifest.json file in a control panel. Here you can define colors, icons and some other settings. Where the images are surved then, is other story.
But this is a whole project by it self and should be discussed more in detail.
I would like to have this in anyway, even if it does not solve everything.